src/main/java/org/vafer/jdeb/ControlBuilder.java

```diff
@@ -103,13 +104,19 @@ void buildControl(BinaryPackageControlFile packageControlFile, File[] controlFil
             if ("conffiles".equals(file.getName())) {
                 foundConffiles = true;
             }
-            
             if (CONFIGURATION_FILENAMES.contains(file.getName()) || MAINTAINER_SCRIPTS.contains(file.getName())) {
                 FilteredFile configurationFile = new FilteredFile(new FileInputStream(file), resolver);
                 configurationFile.setOpenToken(openReplaceToken);
                 configurationFile.setCloseToken(closeReplaceToken);
                 addControlEntry(file.getName(), configurationFile.toString(), outputStream);
 
+            } else if (file.getName().endsWith(".bin")) {
```

Comment:
I'm not aware of an alternative place for the jar: is has to be present when postrm is executed. And at this point dpkg has removed all other files of the package. Even e.g. prerm.

Is there a way to define a variables with my binary content?